lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Good point @anupam128 . I should make sure anything chagrined be rebuilt before running testing. 

I guess adding ProjectReference will be only rebuilt if code is changed, right?